apps/arm/src/main.cpp

```diff
@@ -74,9 +74,55 @@ static mbed_error_status_t setMotionData(CANMsg &msg) {
   return MBED_SUCCESS;
 }
 
-// TODO: Add CAN handler for enabling/disabling current checks.
-// TODO: Add CAN handler for enabling/disabling rpm checks
-// TODO: Add CAN handler for enabling/disabling current checks
+static mbed_error_status_t setSafetyCheck(CANMsg &msg) {
+  HWBRIDGE::CONTROL::SAFETY::SafetyCheckPayload data;
+  msg.getPayload(data);
+
+  const Utility::LookupTable<HWBRIDGE::ARM::ActuatorID, Controller::ActuatorControllerManager *> lut = {
+      {HWBRIDGE::ARM::ActuatorID::TURNTABLE, &Turntable::manager},
+      {HWBRIDGE::ARM::ActuatorID::SHOULDER, &Shoulder::manager},
+      {HWBRIDGE::ARM::ActuatorID::ELBOW, &Elbow::manager},
+      {HWBRIDGE::ARM::ActuatorID::WRISTLEFT, &Wrist::leftManager},
+      {HWBRIDGE::ARM::ActuatorID::WRISTRIGHT, &Wrist::rightManager},
+      {HWBRIDGE::ARM::ActuatorID::CLAW, &Claw::manager}};
+
+  auto act = lut.at(data.actuatorID).value_or(nullptr);
+  if (!act) {
+    return MBED_ERROR_INVALID_ARGUMENT;
+  }
+  auto temp = act->getActiveController();
+
+  switch (msg.getID()) {
+    case HWBRIDGE::CANID::SET_JOINT_CURRENT_CHECK:
+      // auto temp = act->getActiveController();
```

Comment:
remove these comments. Also if you use the ternary operator, you save like 20 lines of code